Assets/Tests/Demo/scripts/verify-replay-via-cli.sh

```
@@ -73,58 +158,71 @@ echo "========================================="
+    echo "========================================="
+    echo "  Recording is active!"
+    echo "  Go to the Unity Game View and play."
+    echo ""
+    echo "  WASD: move | Mouse: rotate"
+    echo "  Left click: red | Right click: blue"
+    echo "  Scroll: scale"
+    echo ""
+    echo "  Press ENTER here when done."
+    echo "========================================="
+    echo ""
+    read dummy
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)